startHbz01ToLobidResources.sh

```diff
@@ -40,6 +40,7 @@ fi
 
 function indexFile() {
 	git checkout $BRANCH
+	git pull
```

Comment:
As the other day, I'd suggest `git pull origin $BRANCH` to avoid different behaviour in different environments.